editor/import/resource_importer_scene.cpp

```diff
@@ -293,16 +293,17 @@ static void _gen_shape_list(const Ref<Mesh> &mesh, List<Ref<Shape3D>> &r_shape_l
 	}
 }
 
-Node *ResourceImporterScene::_fix_node(Node *p_node, Node *p_root, Map<Ref<Mesh>, List<Ref<Shape3D>>> &collision_map, LightBakeMode p_light_bake_mode) {
+Node *ResourceImporterScene::_fix_node(Node *p_node, Node *p_root, Map<Ref<Mesh>, List<Ref<Shape3D>>> &collision_map, LightBakeMode p_light_bake_mode, List<Pair<NodePath, Node *>> &p_node_renames) {
```

Comment:
This should actually be named `r_node_renames` as it's actually mutated by the function (we use `r_` prefix to indicated that it's a mutable reference or "return" argument).